stac_fastapi/pgstac/stac_fastapi/pgstac/types/search.py

```diff
@@ -0,0 +1,113 @@
+"""stac_fastapi.types.search module.
+
+# TODO: replace with stac-pydantic
```

Comment:
Is this todo still relevant? I see you import some `stac_pydantic` classes below